Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -422,7 +422,7 @@ public final class SwiftTargetBuildDescription {
     private func packageNameArgumentIfSupported(with pkg: ResolvedPackage, packageAccess: Bool) -> [String] {
```

Comment:
Can this use the the same code as the extension in `ResolvedPackage`? It seems like we duplicated that now.

Reply:
Good catch, that's addressed now.